doc/architecture/squid.md

```diff
@@ -32,6 +33,29 @@ The goal of this doc is to help a developer build a version of the Squid API in
 * [ocean.agreements.conditions](#ocean.agreements.conditions)
 
 
+## Changelog
+
+You can find older versions of the squid specifications in the [squid-specs folder](squid-specs).
+
+### Squid Spec 0.3 (latest)
+
+* New methods
+  - Get list of the Assets Published by an user
+  - Get list of the Assets Purchased by an user
```

Comment:
Maybe "purchased" isn't the right word because some assets are free. How about "received"?